src/MultiBridgeMessageReceiver.sol

```diff
@@ -238,18 +246,16 @@ contract MultiBridgeMessageReceiver is IMultiBridgeMessageReceiver, ExecutorAwar
                             PRIVATE/INTERNAL FUNCTIONS
     ////////////////////////////////////////////////////////////////*/
 
-    function _updateQuorum(uint64 _quorum) internal {
-        if (_quorum > trustedExecutorsCount() || _quorum == 0) {
-            revert Error.INVALID_QUORUM_THRESHOLD();
-        }
+    function _validateAndUpdateQuorum(uint64 _quorum) private {
```

Comment:
nit: suggest just calling this `_updateQuorum()` for consistency. The fact that validation would also be performed as part of the update operation is implicitly expected based on how we've named other similar functions.

Reply:
Done.